photo_mapper_webserver/photo_gis/api.py

```
@@ -57,35 +57,58 @@ def photo_list(request: HttpRequest, filters:Query[PhotoFilterSchema]):
+    except DuplicatePhotoError:
+        return 400, ErrorResponseSchema(
+            code="DUPLICATE_PHOTO",
+            message="A photo at the same time and location already exists"
+        )
     except ExifException:
-        raise HttpError(400, "Photo is missing datetime or GPS information.")
+        return 400, ErrorResponseSchema(
+            code="DATETIME_MISSING",
+            message="Photo missing datetime or GPS information"
+        )
     except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)